code/drasil-example/Drasil/PDController/DataDefs.hs

```diff
@@ -4,6 +4,8 @@ import Drasil.PDController.Assumptions
 
 import Drasil.PDController.Concepts
 
+import Drasil.PDcontroller.Unitals
```

Comment:
while you're at it, can you delete all those extra blank lines in the import list please?